scripts/pin_the_slow_way.py

```diff
@@ -55,6 +55,7 @@
           'krb5': 'krb5 1.14.*',  # 1.13.2
           'libblitz': 'libblitz 0.10|0.10.*',  # NA
           'libevent': 'libevent 2.0.22',  # 2.0.22
+          'libgdal': 'libgdal 2.1.*',  # 2.1.0
```

Comment:
Looks like the 2.1 series breaks a lot of stuff. Maybe worth pinning tighter.

ref: https://abi-laboratory.pro/tracker/timeline/gdal/